lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -10,7 +10,6 @@ class Base
           ::Errno::ENOENT,
           ::Racc::ParseError,
           ::RubyParser::SyntaxError,
-          ::RuntimeError,
```

Comment:
I don't know: I think we've seen that before, and the suspicion would be somewhere around `FileThreadPool`. I don't think the catch here is involved, since I don't see this file in that stacktrace. Is there a log message near that exception that looks like [the one the catch here logs](https://github.com/codeclimate/codeclimate-duplication/blob/42a53aece30e5f51c251fe2c4e59b501ccb9c225/lib/cc/engine/analyzers/analyzer_base.rb#L24)?